packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -171,11 +171,12 @@ export type VideoPlayerStatus = 'idle' | 'loading' | 'readyToPlay' | 'error';
 
 export type VideoSource =
   | string
+  | number
   | {
       /**
-       * The URI of the video.
+       * The URI of the video or a result of a `require` call for local media sources.
        */
-      uri: string;
+      uri: string | number;
```

Comment:
I'm not convinced to this – a number is not a valid URI 😅 Afaik in `expo-image` the `uri` can only be `string`.

When the result from `require` is passed, it doesn't really makes sense to provide `drm` and `headers`. Maybe the `metadata` should be a player's property instead?